README.md

```diff
@@ -107,4 +107,9 @@ Me trying to make a list of norwegian github orgs
 - https://github.com/tv2norge
 - https://github.com/Sonat-Consulting
 - https://github.com/nettbureau
-- https://github.com/evry
\ No newline at end of file
+- https://github.com/evry
+- https://github.com/visma-prodsec
+- https://github.com/sikkerhet
+- https://github.com/defendable
+- https://github.com/karimelmel
```

Comment:
remove this because this is intended to only be github organzations not individual users